recipes/grass-session/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "grass-session" %}
+{% set version = "0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7155314535790145da8e2e31b0d20cd2be91477d54083a738b5c319164e7f03b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - grass_session
```

Comment:
I think you're going to have to set the `GRASSBIN` environment variable, although I think it can be just a fake path. I'm not sure if there is a simple way to do this with the `imports` section. You'll either need a separate `run_test.py` file to set the `os.environ`, or you may be able to use a `commands` that runs setting the env variable and importing the module using something like:

```
test:
   commands:
      - {{ PYTHON }} -c "import os; os.environ['GRASSBIN'] = 'foo'; import grass_session"
```

The above is untested.

Reply:
I made the recipe at midnight, and I did not realize that the package actually calls the grass binary at import. It will throw an error if given a fake path.
I do not know how to solve this without having a recipe for grass in conda-forge. Sorry for the noise.
What would be best: keeping this PR open and resolving it when grass is in conda-forge, or closing the PR meanwhile?

Reply:
I'm not 100% what the best approach is here. It's a fairly simple library and the tests are just meant to be to check for existence of the package. One possibility is to make a `run_test.py` and attempt to import the library, but then wrap it in a `try/except` and catch the specific exception for the missing binary. That would prove that the package is installed in the test environment and has the anticipated behavior given that `grass` isn't available. This could be changed later if `grass` is available via conda-forge, but I'm not sure that we necessarily want to make `grass` a dependency of `grass-session`, supposing users might want to use a different version of the binary. 

@conda-forge/staged-recipes -- Any thoughts on this?